rest_api/AuthService.adoc

```diff
@@ -0,0 +1,1547 @@
+// Auto-generated by scripts. Do not edit.
+:_mod-docs-content-type: ASSEMBLY
+
+
+
+[id="AuthService"]
+= AuthService
+
+:toc: macro
+:toc-title:
+
+toc::[]
+
+
+
+[id="AuthServiceAddAuthMachineToMachineConfig"]
+== AddAuthMachineToMachineConfig
+
+`POST /v1/auth/m2m`
+
+AddAuthMachineToMachineConfig creates a new auth machine to machine config.
+
+=== Description
+
+
+
+
+
+
+
+=== Parameters
+
+
+==== Body Parameter
+
+[cols="2,3,1,1,1"]
+|===
+|Name| Description| Required| Default| Pattern
+
+| body
+|  <<V1AddAuthMachineToMachineConfigRequest>>
+| X
+|
+|
+
+|===
+
+
+
+
+
+=== Return Type
+
+<<V1AddAuthMachineToMachineConfigResponse>>
+
+
+=== Content Type
+
+* application/json
+
+=== Responses
+
+.HTTP Response Codes
+[cols="2,3,1"]
+|===
+| Code | Message | Datatype
+
+
+| 200
+| A successful response.
+|  <<V1AddAuthMachineToMachineConfigResponse>>
+
+
+| 0
+| An unexpected error response.
+|  <<RuntimeError>>
+
+|===
+
+=== Samples
+
+
+
+
+
+
+
+
+
+ifdef::internal-generation[]
+=== Implementation
+
+
+
+endif::internal-generation[]
+
+
+[id="AuthServiceDeleteAuthMachineToMachineConfig"]
+== DeleteAuthMachineToMachineConfig
+
+`DELETE /v1/auth/m2m/{id}`
+
+DeleteAuthMachineToMachineConfig deletes the specific auth machine to machine config. In case a specified auth machine to machine config does not exist is deleted, no error will be returned.
+
+=== Description
+
+
+
+
+
+
+
+=== Parameters
+
+==== Path Parameters
+
+[cols="2,3,1,1,1"]
+|===
+|Name| Description| Required| Default| Pattern
+
+| id
+|
+| X
+| null
+|
+
+|===
+
+
+
+
+
+
+=== Return Type
+
+
+<<Object>>
```

Comment:
These are showing up as clickable links but don't go anywhere when clicked - is that the behavior that is supposed to happen? 
![Screenshot 2024-09-25 at 3 20 10 PM](https://github.com/user-attachments/assets/a698fe15-c684-4318-b8ae-50494c595f1b)